src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,113 @@
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { format } from 'date-fns';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ExportOptions } from '../lib/import-export/export/types';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+const getFileName = ( selectedSite: SiteDetails ) => {
+	const timestamp = format( new Date(), 'yyyy-MM-dd-HH-mm-ss' );
+	return `studio-backup-${ selectedSite.name
+		.toLocaleLowerCase()
```

Comment:
What do you think about using `sanitizeFolderName` for that?

Reply:
I updated the PR and used that ( wasn't aware of it ).
However:
- I moved that function at `/lib/generate-site-name` ( as I didn't want to create another file for utility functions and it's kinda related )
- I re-exported it via `fs-utils`
The reason is that there were places that used that function via `fs-utils`  ( which makes sense for that function to be there ), however the `fs-utils` imports `path`, and `path` it's not available in the renderer.
`generate-site-name`  is not using any node libraries, and also the sanitize function, uses just js functions, so by moving it there, I was able to import it, and use it, in renderer.